.travis.yml

```diff
@@ -1,4 +1,10 @@
 language: d
+os:
+  - linux
+  - osx
```

Comment:
Yes, but that error will be completely unrelated to the pull request that triggers it, and that is unfortunate. If Travis does not support D on OS X, the `os:` section should just be removed.